src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,9 +33,10 @@ public class RobotContainer {
   
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
-
   private static BatterySubsystem batterySubsystem;
-
+  private static LedSubsystem ledSubsystem = new LedSubsystem();
```

Comment:
Why is this subsystem declared as static?

Reply:
Was giving me null when I tried to simulate it. Fixed after doing the changes I marked as resolved. I don't know how or why.